text/0005-architecture-reviews.md

```diff
@@ -0,0 +1,135 @@
+  * Someone to **chair** the review (for a Roadmap Review this will be the *least* senior person in the management chain with the ability to provide resources to the problem, for a Design Review this will be the *most* experienced engineer outside of the team willing to put time into the problem. As engineers don’t review their own code they shouldn’t review their own designs)
+  * The **question** (or questions) to be resolved, which should be agreed between the team asking for review and the chair. For a Roadmap Review this is likely to be in the form of a Go/No-Go decision. For a Design Review this is likely to be a more open question (e.g. "How should this design be improved")
+
+2. The team produces a **Review Packet** designed to answer the question identified in step 1. The production of the Review Packet is expected to be collaborative and iterative to ensure that the review focuses on the key questions and the team presents its best case. This is particularly true of a Roadmap Review where the consequences of the answer are likely to be higher.
+The Review Packet includes:
+  * A lay summary of the problem space and the stakeholders which focused on defining a shared language and identifying the key forces behind the problem
+  * A brief explaining what the team proposes which should read more like an encyclopedia entry than a marketing document.  The brief should identify:
+    - the forces on the system
+    - the proposed system’s essential architecture
+    - scenarios that exercise the relevant forces against the architecture
+    - how the proposed system handles these scenarios
+  * A competitive analysis suggesting alternatives, costs, and opportunities.
+
+3. The chair creates a list of questions to be discussed during the review. These questions should be:
+  * Open-ended
+  * Made available to the the team prior to the review so they can prepare answers
+
+4. The chair convenes a review meeting. The discussions of this meeting should be carefully minuted. The meeting comprises:
+  * The team making the proposal
+  * The chair
+  * (For Design Reviews) other senior engineers who will provide valuable input.
+
+The review process should address the question(s) where possible, but the meeting may also decide to alter the the question(s) and undergo a subsequent review. For questions that are go/no-go in nature the decision does not need to be taken during the meeting, although we should be very open about conversations that happen after the review meeting.
+
+The ideal review process scales well. Ideally the same basic system should work for a quick 2 person decision over the best way to design a certain feature, and for a critical organization wide decision about a path to take. The term “team” is used above although we strongly recommend design reviews for smaller questions. If you feel yourself wondering if some design is best, it should be easy to valuable to perform a review.
```

Comment:
The continuing repetition of ideal makes it sound as if you don't have confidence in this. Perhaps more positive/definite phrasing?